images/fcos-bastion-image/root/usr/lib/systemd/system/squid-proxy.service

```diff
@@ -0,0 +1,32 @@
+[Unit]
+Description=Creates the squid proxy container
+Wants=network-online.target
+After=network-online.target
+RequiresMountsFor=%t/containers
+
+[Service]
+Environment=PODMAN_SYSTEMD_UNIT=%n
+EnvironmentFile=/usr/share/bastion_services/squid/squid_systemd.conf
+Restart=on-failure
+TimeoutStopSec=70
+ExecStartPre=/bin/rm -f %t/%n.ctr-id
+ExecStartPre=/usr/bin/podman login quay.io
+ExecStartPre=-/bin/sh -c '/usr/bin/mkdir -p ${PODMAN_SQUID_BASEDIR}/{cache,log,lib}'
+ExecStartPre=-/bin/sh -c '/usr/bin/podman run --rm --interactive -v "${PODMAN_SQUID_BASEDIR}/lib:/var/lib:Z" ${PODMAN_SQUID_IMAGE} bash < /usr/share/bastion_services/squid/create_ssl_db.sh'
+ExecStartPre=-/bin/sh -c '/usr/bin/chown -R ${PODMAN_SQUID_UID}:${PODMAN_SQUID_UID} ${PODMAN_SQUID_BASEDIR}'
+ExecStartPre=-/bin/sh -c 'podman unshare chown ${PODMAN_SQUID_UID}:${PODMAN_SQUID_UID} -R ${PODMAN_SQUID_BASEDIR}'
+ExecStart=/usr/bin/podman run --rm --log-driver=journald -d --name squid-proxy --hostname squid-proxy --net host --user ${PODMAN_SQUID_UID} \
+    -v "/usr/share/bastion_services/squid:/etc/squid:Z" \
+    -v "${PODMAN_SQUID_BASEDIR}/cache:/var/spool/squid:Z" \
+    -v "${PODMAN_SQUID_BASEDIR}/log:/var/log/squid:Z" \
+    -v "${PODMAN_SQUID_BASEDIR}/lib:/var/lib:Z" \
+    ${PODMAN_SQUID_IMAGE}
+ExecStop=/usr/bin/podman stop --ignore --cidfile=%t/%n.ctr-id
+ExecStopPost=/usr/bin/podman rm -f --ignore --cidfile=%t/%n.ctr-id
+ExecStopPost=-/bin/sh -c 'rm -rf ${PODMAN_SQUID_BASEDIR}/lib/ssl_db'
```

Comment:
Do we need deleting the /var/lib/ssl_db folder or can we rely on the initialization done only once.